spec/api/comment_thread_spec.rb

```diff
@@ -263,15 +263,15 @@ def move_to_front(ary, *vals)
             t5c.update(body: "changed!")
             t5c.save!
             actual_order = thread_result_order("activity", "desc")
-            expected_order = move_to_front(@default_order, "t5")
+            expected_order = @default_order
```

Comment:
Yes, we are not updating `last_activity_at` on thread or comment update actions.
No, there were no tests to check sorting on creation. I have added a few in my latest commit. Kindly review them too.